daemon/containerd/image_pull.go

```diff
@@ -104,6 +104,10 @@ func (i *ImageService) PullImage(ctx context.Context, image, tagOrDigest string,
 	infoHandler := snapshotters.AppendInfoHandlerWrapper(ref.String())
 	opts = append(opts, containerd.WithImageHandlerWrapper(infoHandler))
 
+	// Allow pulling application/vnd.docker.distribution.manifest.v1+prettyjws images
+	// by converting them to OCI manifests.
+	opts = append(opts, containerd.WithSchema1Conversion)
```

Comment:
do you know if we can detect these in this flow, and print a deprecation warning? (see https://github.com/moby/moby/pull/46137)

Reply:
We could print that deprecation in one of the image handlers maybe? That's if the manifest is not already converted once it gets to the handler

Reply:
Yes, we can! I somehow missed the deprecation warning when looking at the non-c8d output 😄